pinot-core/src/main/java/org/apache/pinot/core/upsert/PartitionUpsertMetadataManager.java

```diff
@@ -85,8 +97,7 @@ public ThreadSafeMutableRoaringBitmap addSegment(String segmentName, Iterator<Re
               // committing a consuming segment, or reloading a completed segment.
 
               // Update the record location when the new timestamp is greater than or equal to the current timestamp.
-              // Update the record location when there is a tie because the record locations should point to the new
-              // segment instead of the old segment being replaced. Also, do not update the valid doc ids for the old
+               // segment instead of the old segment being replaced. Also, do not update the valid doc ids for the old
```

Comment:
This line is mis-deleted?

Reply:
no.. auto formatting

Reply:
This is not auto formatting. One line is mis-deleted. Please double check